RemoteHealthcare/BikeManager.cs

```diff
@@ -92,9 +87,9 @@ public bool CloseConnections()
             return false;
         }
 
-        private static void BleBike_SubscriptionValueChanged(object sender, BLESubscriptionValueChangedEventArgs e)
+        private static void BleBike_SubscriptionValueChanged(object sender, avansBikeData e)
```

Comment:
Zie [Style Guide](https://github.com/Ti-2-1-RH-A1/remote-healthcare/blob/development/.github/STYLE_GUIDELINES.md#methodes) voor fout bij Methode structuur fout (Danwel hoe methode is opgebouwd of commentaar)